features/capybara_javascript_drivers.feature

```diff
@@ -8,13 +8,13 @@ Feature: Capybara Javascript Drivers
       """
       @javascript
       Feature: Create appointments
-        Scenario: Visit the Constitution on May 17
+        Scenario: Create an appointment using the Web Interface
```

Comment:
I've cherry-picked @luke-hill's commit to fix some failing features.

@luke-hill you may find commit af590eb useful on your branch as I've followed your lead and fixed a couple more.